### PR TITLE
disable running auto scaling related e2e cases

### PR DESCRIPTION
**Pull request by dragonly**
### What problem does this PR solve?

disable running auto scaling related e2e cases for e2e testing stability.

### What is changed and how does it work?

disable running auto scaling related e2e cases

### Code changes

- [x] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
None
```

### CODE REVIEW

File: tests/e2e/tidbcluster/tidbcluster.go [hunk 1]
```diff
@@ -932,7 +932,8 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 		})
 	})
 
-	ginkgo.Context("[Feature: AutoScaling]", func() {
+	// add [Stability] to prevent running auto scaling related cases for now
+	ginkgo.Context("[Stability][Feature: AutoScaling]", func() {
```
Comment (DanielZhangQD) [inline tests/e2e/tidbcluster/tidbcluster.go:936]: Suggest moving to the stability test files.

Comment (dragonly) [reply tests/e2e/tidbcluster/tidbcluster.go:936]: fixed in next commit

File: tests/e2e/tidbcluster/tidbcluster.go [hunk 2]
```diff
@@ -114,8 +111,8 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 		framework.ExpectNoError(err, "failed to create port forwarder")
 		fwCancel = cancel
 		cfg = e2econfig.TestConfig
-		OperatorFeatures := map[string]bool{"AutoScaling": true}
-		cfg.OperatorFeatures = OperatorFeatures
+		// OperatorFeatures := map[string]bool{"AutoScaling": true}
```
Comment (csuzhangxc) [inline tests/e2e/tidbcluster/tidbcluster.go:114]: How about just removing these lines?

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: tests/e2e/tidbcluster/tidbcluster.go (hunk 2 above)
Comment (dragonly) [reply tests/e2e/tidbcluster/tidbcluster.go:114]: This will make the auto scaling case here confusing

**Pull request merged**